cmake/toolchain.cmake

```diff
@@ -12,6 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Just pointing out I cannot remember why I originally put this here. I think it might have been to make linking in shared objects easier, but really cannot remember.